enterprise/views.py

```diff
@@ -1009,32 +1059,41 @@ def get_program_details(self, request, program_uuid):
 
         * Take the program UUID and get specific details about the program.
         * Determine whether the learner is enrolled in the program.
+        * Determine whether the learner is certificate eligible for the program.
         """
         try:
-            program_details = CourseCatalogApiServiceClient().get_program_by_uuid(program_uuid)
+            course_catalog_api_client = CourseCatalogApiServiceClient()
         except ImproperlyConfigured:
             raise Http404
 
+        program_details = course_catalog_api_client.get_program_by_uuid(program_uuid)
         if program_details is None:
             raise Http404
 
+        program_type_slug = program_details['type'].lower().replace(' ', '-')
```

Comment:
Just realized I should be using `django.utils.text.slugify` here. Will do it on the next push after initial code review.